### PR TITLE
fix for Basemap.pcolormesh 'ortho' projection

### PR DESCRIPTION
**Pull request by 2sn**
fix for #470 as discussed in comment there

### CODE REVIEW

**Comment by 2sn** [comment]
*It seems to be working with current numpy; irrespective, the aim of the
code as written is that the dimensions match.*

*Thanks for all the suggestions.  Do I need to do a new pull request you
you will integrate a version with your modifications?*

File: lib/mpl_toolkits/basemap/__init__.py [hunk 1]
```diff
@@ -3426,6 +3426,27 @@ def pcolormesh(self,x,y,data,**kwargs):
...
+                (x[:-1,1:] > 1e20) |
+                (x[1:,1:] > 1e20) |
+                (y[:-1,:-1] > 1e20) |
+                (y[1:,:-1] > 1e20) |
+                (y[:-1,1:] > 1e20) |
+                (y[1:,1:] > 1e20)
+                )
+            # we do not want to overwrite original array
+            data = data[:nx-1,:ny-1].copy()
+            data[mask] = np.nan
```
[the remark this answers is not in the document]
Comment (2sn) [reply lib/mpl_toolkits/basemap/__init__.py]: This is why I make a copy of slice of data with the correct dimensions: `data = data[:nx-1,:ny-1].copy()`

**Comment by ricitron** [comment]
Is it still possible to merge this fix?

**Comment by molinav** [comment]
Hi @2sn, you do not need to create a new pull request, you can just create a new commit in your `master` branch with the reviewed modifications and I guess the CI pipeline will just get triggered again. After Travis passes the build I can merge the pull request.

**Comment by 2sn** [comment]
@molinav thanks for your follow-up

My apologies, I have since transitioned my code to `cartopy`, although it was slightly painful to learn how it works.

**Comment by molinav** [comment]
No worries @2sn, then I will just merge the pull request as it is and I will apply the review changes afterwards in `master`. There should be no problem with the CI, I think your pull request originally failed because the Travis configuration was quite outdated. Thanks for your contribution!

**Pull request merged**